Makefile

```diff
@@ -1,5 +1,39 @@
+# Code generated by Mage-tools. DO NOT EDIT.
```

Comment:
minor: let's have blank lines in the generated file that will make it indistinguishable from a hand-written file - i.e. blank line after this top-level comment, and a blank line after `.DEFAULT_GOAL`

Reply:
Fixed